Rdmp.UI/Rdmp.UI.csproj

```diff
@@ -11,6 +11,10 @@
     <LangVersion>7.1</LangVersion>
     <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Can we get this on all csproj? maybe outside the PropertyGroup tag too?

Reply:
I thought it was set to "Portable" on purpose to cope with Linux builds... but ok I can put it in for all csproj.